### ISSUE TITLE
Port twisted.protocols.memcache to Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8726]
|[<img alt="charmander's avatar" src="https://avatars.githubusercontent.com/u/1889843?s=50" width="50" height="50">](https://github.com/charmander)| @charmander reported|
|-|-|
|Trac ID|trac#8726|
|Type|enhancement|
|Created|2016-08-05 06:16:03Z|

As mentioned on the PR, just one or two minor things, then I think we're good to merge :)

<details><summary>Searchable metadata</summary>

```
trac-id__8726 8726
type__enhancement enhancement
reporter__charmander charmander
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__charmander charmander
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470377763714778 1470377763714778
changetime__1470817782138468 1470817782138468
version__None None
owner__charmander charmander

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="charmander's avatar" src="https://avatars.githubusercontent.com/u/1889843?s=50" width="50" height="50">](https://github.com/charmander)<a name="note_2"></a>|@charmander commented|
|-|-|

https://github.com/twisted/twisted/pull/464

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)|@hawkowl set owner to @charmander|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|GitHub set status to `closed`|
|-|-|

In changeset 28a8dc4249e522cc74dfab4d09b7da4730cd9a55
```
#!CommitTicketReference repository="" revision="28a8dc4249e522cc74dfab4d09b7da4730cd9a55"
Merge charmander:memcache-py3-8726: Port twisted.protocols.memcache to Python 3

Author: charmander
Reviewers: adiroiban, rodrigc, hawkowl
Fixes: #8726
```
